example/http-mux/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"net/http"
+
+	_ "github.com/swaggo/http-swagger/example/http-mux/docs" // docs is generated by Swag CLI, you have to import it.
+
+	httpSwagger "github.com/swaggo/http-swagger/v2"
+)
+
+// @title Swagger Example API
+// @version 1.0
+// @description This is a sample server Petstore server.
+// @termsOfService http://swagger.io/terms/
+
+// @contact.name API Support
+// @contact.url http://www.swagger.io/support
+// @contact.email support@swagger.io
+
+// @license.name Apache 2.0
+// @license.url http://www.apache.org/licenses/LICENSE-2.0.html
+
+// @host petstore.swagger.io
+// @BasePath /
+func main() {
+	r := http.NewServeMux()
+
+	r.Handle("GET /swagger/", httpSwagger.Handler(
```

Comment:
Did yiy test the r.Handle route ? Looks to me like invalid according to the documentation

```
import (
	"fmt"
	"net/http"
)

type apiHandler struct{}

func (apiHandler) ServeHTTP(http.ResponseWriter, *http.Request) {}

func main() {
	mux := http.NewServeMux()
	mux.Handle("/api/", apiHandler{})
	mux.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
		// The "/" pattern matches everything, so we need to check
		// that we're at the root here.
		if req.URL.Path != "/" {
			http.NotFound(w, req)
			return
		}
		fmt.Fprintf(w, "Welcome to the home page!")
	})
}
````